content/updates/2020-05/3-visibility-on-jobs.md

```diff
@@ -0,0 +1,10 @@
+# More visibility on PIM jobs
+::: meta-data type="Improvement" features="Productivity, Process tracker" available="mid-May" link-to-doc="../articles/monitor-jobs.html#how-to-monitor"
+:::
+
+To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker`. It will enable you to see all jobs of any status under the Process Tracker. You will now be able to check if you have any ongoing running jobs and which users launched them . :wink:
```

Comment:
```suggestion
To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker`. It will enable you to see all jobs of any status under the Process Tracker. You will now be able to see the ongoing or previous PIM jobs and if needed which users launched them. :wink:
```